### PR TITLE
GFORMS-2929 - Show confirmation choice on the CYA page by default but have choice to hide if not required

### PR DESCRIPTION
**Pull request by cmanson**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -17,6 +17,7 @@ object Dependencies {
     "uk.gov.hmrc"                 %% "domain-play-30"             % "9.0.0",
     "org.playframework"           %% "play-json-joda"             % "3.0.2",
     "org.julienrf"                %% "play-json-derived-codecs"   % "11.0.0",
+    "ai.x"                        %% "play-json-extensions"       % "0.42.0",
```
Comment (josef-vlach) [inline project/Dependencies.scala:20]: Why we need this dependency?

Comment (cmanson) [reply project/Dependencies.scala:20]: So we can format case class larger than 22 arity (one new member on FormComponent takes it to 23)

Comment (josef-vlach) [reply project/Dependencies.scala:20]: We have 
```
"org.julienrf"                %% "play-json-derived-codecs"   % "11.0.0",
```
for that

Comment (cmanson) [reply project/Dependencies.scala:20]: Ah okay, I will change it

**Review by josef-vlach — APPROVED** [review]

**Pull request merged**